archaius-core/src/main/java/netflix/archaius/CascadeStrategy.java

```diff
@@ -0,0 +1,29 @@
+package netflix.archaius;
```

Comment:
It's only completely unnecessary if you have a monopoly on all of the possible domains. ;-)